File: entity-recognition/google-cloud-natural-language/GoogleCloudNaturalLanguageEntityRecognition.java
// "content" is a string containg the input document.

try (LanguageServiceClient language = LanguageServiceClient.create()) {
    Document document = Document.newBuilder()
        .setContent(content)
        .setType(Document.Type.PLAIN_TEXT)
        .build();

    List<Entity> entities = language
        .analyzeEntities(document, EncodingType.UTF8)
        .getEntitiesList();
}

File: entity-recognition/google-cloud-natural-language/GoogleCloudNaturalLanguageEntityRecognition.kt
val client = LanguageServiceClient.create()

val document = Document.newBuilder()
    .setContent(text)
    .setType(Document.Type.PLAIN_TEXT)
    .build()

val entities = client
    .analyzeEntities(document, EncodingType.UTF8)
    .entitiesList
